### PR TITLE
Time-tiling for Devito

### PR DESCRIPTION
**Pull request by georgebisbas**
Initializing time-tiling work for Devito.

### CODE REVIEW

File: tests/test_dse.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 pytestmark = skipif(['yask', 'ops'])
 
+# Tests for DLE will be needed
```
Comment (btrb) [inline tests/test_dse.py]: you meant DSE?

Comment (georgebisbas) [reply tests/test_dse.py]: Right!

**Comment by FabioLuporini** [comment]
This is clearly premature for a PR?

**Comment by georgebisbas** [comment]
> This is clearly premature for a PR?

Yes it is. Do you mind if I keep it as a PR for non local testing?

**Comment by FabioLuporini** [comment]
you don't need a PR. On github, you can go to your branch, then go to the list of commits, and you'll see that the latest pushed commit lets you access the CI report , see for example [here](https://github.com/opesci/devito/commits/time-tiling)

**Comment by georgebisbas** [comment]
THanks, I did not know that, cause it is the first PR as opesci member and not from the fork. Temporarily closed then.

**Pull request closed without merging**